HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -83,6 +85,10 @@
 #define SYSTEM_AUDIO_MODE_OFF 0x00
 #define AUDIO_DEVICE_POWERSTATE_OFF 1
 
+#define DEFAULT_VIDEO_LATENCY 100
```

Comment:
Are these value defiend somewhere else? What if thosd define gets updated?

Reply:
For now, default values are not implemented on TV Server side. Latency Info will be updated only when there is a change in latency and hence here is the only place where default values are being handled